### PR TITLE
Added str method to  models

### PR DESCRIPTION
**Pull request by JulienSchoener**

### CODE REVIEW

File: petoshield_api/apps/pet/models.py [hunk 1]
```diff
@@ -35,4 +38,7 @@ class Pet(BaseModel):
     gender = models.CharField(max_length=1, choices=PET_GENDER)
     species = models.CharField(max_length=3, choices=PET_SPECIES)
     breed = models.ForeignKey(Breed, on_delete=models.SET(get_default_breed), related_name='pets')
-    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE, related_name='pets')
\ No newline at end of file
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE, related_name='pets')
+    
+    def __str__(self):
+        return f"{self.name} ({self.species})"
```
Comment (vitaliisili) [inline petoshield_api/apps/pet/models.py:44]: Return only the name of the pet

**Review by vitaliisili — CHANGES_REQUESTED** [review]
Add also __str__ for policy app models like serviceProvider

File: petoshield_api/apps/pet/models.py (hunk 1 above)
Comment (JulienSchoener) [reply petoshield_api/apps/pet/models.py]: updated

**Review by JulienSchoener — COMMENTED** [review]
check

File: petoshield_api/apps/policy/models.py [hunk 2]
```diff
@@ -46,11 +52,17 @@ class InsuranceCase(BaseModel):
...
+    def __str__(self):
+        return self.claim_date
 
 class IncomingInvoice(BaseModel):
     invoice_date = models.DateField()
     amount = models.DecimalField(max_digits=8, decimal_places=2)
     insurance_case = models.ForeignKey(InsuranceCase, on_delete=models.CASCADE, related_name='incoming_invoice')
+    
+    def __str__(self):
+        return self.invoice_date
```
Comment (vitaliisili) [inline petoshield_api/apps/policy/models.py:65]: Here should be invoice amount

**Review by vitaliisili — CHANGES_REQUESTED** [review]
In policy IncomingInvoice should return invoice anount instead of invoice date

File: petoshield_api/apps/policy/models.py (hunk 2 above)
Comment (JulienSchoener) [reply petoshield_api/apps/policy/models.py]: applied changes

**Review by vitaliisili — APPROVED** [review]

**Pull request merged**